### PR TITLE
Disk API - list and rescan

### PR DESCRIPTION
**Pull request by kkmsft**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test

/kind feature

> /kind flake

**What this PR does / why we need it**:
Disk API - list and rescan

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: client/api/volume/v1alpha1/api.pb.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
-// source: api.proto
+// source: github.com/kubernetes-csi/csi-proxy/client/api/volume/v1alpha1/api.proto
```
Comment (ddebroy) [inline client/api/volume/v1alpha1/api.pb.go]: Can you separate out the PB generation for volume APIs into a separate PR please (and maybe open that PR directly against the master repo)?

Comment (kkmsft) [reply client/api/volume/v1alpha1/api.pb.go]: Removed.

**Comment by kkmsft** [comment]
Have rebased the PR to have four functionally separated out commits. PTAL.

**Comment by ddebroy** [comment]
lgtm - thanks!

**Pull request merged**